chef_master/source/chef_overview.rst

```diff
@@ -103,7 +105,7 @@ The workstation is where users do most of their work, including:
 
 .. end_tag
 
-The Chef Development Kit tooling encourages integration and unit testing, and defines workflow around cookbook authoring and policy, but it's important to note that you know best about how your infrastructure should be put together. Therefore, Chef makes as few decisions on its own as possible. When a decision must be made tools uses a reasonable default setting that can be easily changed. While Chef encourages the use of the tooling packaged in the Chef DK, none of these tools should be seen as a requirement or pre-requisite for being successful using Chef.
+The Chef Development Kit tooling encourages integration and unit testing, and defines workflow around cookbook authoring and policy, but it's important to note that you know best about how your infrastructure should be put together. Therefore, Chef makes as few decisions on its own as possible. When a decision must be made tools uses a reasonable default setting that can be easily changed. While Chef encourages the use of the tooling packaged in the ChefDK, none of these tools should be seen as a requirement or pre-requisite for being successful using Chef.
```

Comment:
I feel like this statement should reference workstation. but also it's wrong to imply we don't have opinions. With workstation we are being opinionated with lots of stuff.

Reply:
Here's what I suggest: Delete the paragraph and move the content of the note here. Don't use the note elsewhere on this page.

Reply:
done.